.github/workflows/docs.yml

```diff
@@ -2,26 +2,27 @@ name: Docs
 
 on:
   push:
-    branches: [ master, develop, 'dev/*' ]
+    branches: [ '**' ]
   pull_request:
-    branches: [ master, develop ]
+    branches: [ '**' ]
```

Comment:
Docs should only really be built when attempting to merge to master. I don't think this should be a wildcard.